test/e2e/atlas/integrations_test.go

```diff
@@ -50,6 +50,7 @@ func TestIntegrations(t *testing.T) {
 			integrationsEntity,
 			"create",
 			datadogEntity,
+			"--debug",
```

Comment:
are we keeping this?

Reply:
nope, that was me testing, sorry! removing